colin-api/src/colin_api/version.py

```diff
@@ -22,4 +22,4 @@
 Development release segment: .devN
 """
 
-__version__ = '2.44.1'  # pylint: disable=invalid-name
```

Comment:
I think we can leave colin api out of this release.  @kialj876 release the latest colin api changes to prod already last week https://github.com/bcgov/lear/actions/runs/3290995074

Reply:
thanks